.changeset/eleven-bees-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/graph': major
+---
+
+The `adjacencyMapToArray(…)` helper function has been introduced, which converts an adjacency map to an array of `{ state, event, nextState }` objects.
```

Comment:
Might be a good place to add a simple use case example. I don't really know what this means or how I might use it.

Reply:
Added an example